src/ExternalSearch.Providers.DuckDuckgo/Vocabularies/DuckDuckGoOrganizationVocabulary.cs

```diff
@@ -42,6 +42,14 @@ public DuckDuckGoOrganizationVocabulary()
 
             // Results
             this.Websites         = this.Add(new VocabularyKey("websites"));
+
```

Comment:
Do we need this now that we have the dynamic vocabulary being created by calling the vocabulary repository by reflection?

Reply:
related topics vocabulary keys now create dynamically